test/embedding/test-embedding.js

```diff
@@ -13,31 +13,40 @@ if (common.isWindows) {
 }
 binary = path.resolve(__dirname, '..', '..', binary);
 
+const env = {
+  ...process.env,
+  NODE_DISABLE_COLORS: true
```

Comment:
What made these changes necessary?

Reply:
https://github.com/nodejs/node/blob/f1940824d9f0e769d4ecce07307d4b126cd8dbb4/lib/internal/tty.js#L182 was picking up iTerm env variables in my shell.

Reply:
this has been pulled since the console changes got pulled